### ISSUE TITLE
Add learning rate finder.

### ISSUE THREAD

**Issue by Serhiy-Shekhovtsov** [Serhiy-Shekhovtsov/mars-anomalies-aae#5]
We can find an optimal learning rate using the approach from the paper: **Cyclical Learning Rates for Training Neural Networks**

I think we can use this library: https://github.com/davidtvs/pytorch-lr-finder or look for a similar one.
